giskard/models/base/model.py

```diff
@@ -38,6 +37,23 @@
 logger = logging.getLogger(__name__)
 
 
+def _validate_text_generation_params(name, description, feature_names):
+    if not name or not description:
+        raise ValueError(
+            "The parameters 'name' and 'description' are required for 'text_generation' models, please make sure you "
+            "passed them when wrapping your model. Both are very important for the LLM-assisted testing and scan to "
+            "work properly. Name and description should briefly describe the expected behavior of your model. Check "
+            "our documentation for more information."
+        )
+
+    if not feature_names:
```

Comment:
explicit check for `None`

Reply:
This is something I do intentionally (I don’t want empty feature_names, even if it's just `[]`), but you’re right that it might break with `np.array`